client/src/components/Atoms/Layout.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+export const CenterRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+`
+
+export const CenterColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`
+
+export const ButtonRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+`
```

Comment:
Curious to hear your thoughts about this approach. I was trying to abstract some common cases while also still maintaining some specificity (vs just making a `Flex` component that takes all the flex props). Not sure what the best approach is

Reply:
Yeah, I also went through the same debates. Here's my thinking of a way to approach this:
- Make an opinionated component based on the cases we think are most common
- Use inline styles to override when necessary
- If we see a common override, create a new component or add a prop

Reply:
Updated to `Row` and `Column` elements here: [b8d4b0e](https://github.com/votingworks/arlo/pull/1638/commits/b8d4b0e65c14fb718f9b09d161686ddb0f714f30)